res2df/wcon.py

```diff
@@ -80,7 +80,7 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
             fill with arguments
     """
     parser.add_argument(
-        "DATAFILE", help="Name of Eclipse DATA file or Eclipse include file."
+        "DATAFILE", help="Name of reservoir .DATA file or resdata include file."
```

Comment:
parser.add_argument(
"DATAFILE", help="Name of the .DATA input file for the reservoir simulator."